shacl12-core/index.html

```diff
@@ -1702,7 +1703,7 @@ <h3>Node Expressions</h3>
 					The evaluation may also result in an <dfn>evaluation failure</dfn>.
 				</div>
 				<p>
-					This SHACL Core specification only defines exactly the <a>node expression functions</a> from the following two sections.
+					This SHACL Core specification only defines exactly the <a>node expression functions</a> from the following two subsections.
```

Comment:
makes the `exactly` apply more clearly to `defines` than to `the ... functions ...`
```suggestion
					This SHACL Core specification only exactly defines the <a>node expression functions</a> from the following two subsections.
```

Reply:
Resolved but not applied, and no discussion?